knplabs/knp-snappy-bundle/1.4/etc/packages/knp_snappy.yaml

```diff
@@ -0,0 +1,10 @@
+# Read Documentation: https://github.com/KnpLabs/KnpSnappyBundle/blob/v1.4/README.markdown
+knp_snappy: ~
+#    temporary_folder: %kernel.cache_dir%/snappy
+#    pdf:
+#        enabled:    true
+#        options: []
+#    image:
+#        enabled:    true
+#        options: []
```

Comment:
@stof I totally agree. 
Some say that you need documentation for a better practice. [e.g](https://github.com/symfony/recipes-contrib/pull/39/files/853054988bd6de2a13835898a0de38019c4ab028#r117183291) 
This topic deserves more clarity in your documentation.

What do you think we should do?
```
# Read Documentation: https://github.com/KnpLabs/KnpSnappyBundle/blob/v1.4/README.markdown
knp_snappy: ~

```
Just that ?